### PR TITLE
Update user-manual.md - Settings Screen

### PR DESCRIPTION
**Pull request by justbruce**
Add 2 additional methods to reach the settings screen as suggested by @sspanak

### CODE REVIEW

**Review by justbruce — COMMENTED** [review]
corrected the Default settings screen key to "Hold *"

File: docs/user-manual.md [hunk 1]
```diff
@@ -103,5 +103,14 @@ Backspace.
 On the Settings screen, you can choose languages for typing, configure the keypad hotkeys or change the application appearance.
 
 To access it:
+
+#### Method 1
 - Start typing in a text field to wake up TT9.
 - Use the on-screen gear button or press the Settings Key.
+
+#### Method 2
+- Press the Assigned Hotkey [Default: Hold *]
```
Comment (sspanak) [inline docs/user-manual.md]:
```suggestion
- Start typing in a text field to wake up TT9.
- Use the assigned hotkey [Default: Hold ✱].
```

File: docs/user-manual.md [hunk 2]
```diff
@@ -103,5 +103,14 @@ Backspace.
...
+
+#### Method 3
+- Thru Android Settings: System -> Languages -> (On-screen) Keyboard(s) -> Traditional T9.
```
Comment (sspanak) [inline docs/user-manual.md]: Let's extend this a bit:

```suggestion
- Go to Android Settings → System → Languages → Keyboards (or On-Screen Keyboards). This is where all installed keyboards are configured.
- Select "Traditional T9".

_The actual menu names may vary depending on your phone, Android version and language._
```

File: docs/user-manual.md [hunk 3]
```diff
@@ -103,5 +103,14 @@ Backspace.
 On the Settings screen, you can choose languages for typing, configure the keypad hotkeys or change the application appearance.
 
 To access it:
```
Comment (sspanak) [inline docs/user-manual.md]:
```suggestion
### How to access the Settings?
```

**Review by sspanak — CHANGES_REQUESTED** [review]

**Comment by sspanak** [comment]
I actually already fixed it, but your version seems to be shorter and more clear. Let's use a mixture of both.

File: docs/user-manual.md [hunk 4]
```diff
@@ -102,6 +102,19 @@ Backspace.
 ## Settings Screen
 On the Settings screen, you can choose languages for typing, configure the keypad hotkeys or change the application appearance.
 
-To access it:
+### How to access the Settings?
+
+#### Method 1
 - Start typing in a text field to wake up TT9.
 - Use the on-screen gear button or press the Settings Key.
```
Comment (sspanak) [inline docs/user-manual.md]:
```suggestion
- Use the on-screen gear button.
```

**Review by sspanak — APPROVED** [review]

**Pull request merged**